kfs-core/src/main/java/edu/arizona/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImpl.java

```diff
@@ -79,7 +106,7 @@ public void setProcurementCardGroupService(GroupService procurementCardGroupServ
     public boolean rerouteProcurementCardDocuments() {
         
         // check for reconciler group rerouting
-        boolean groupRerouting = parameterService.getParameterValueAsBoolean(ProcurementCardRerouteDocumentsStep.class, REROUTE_PCDO_DOCUMENTS_IND_PARM_NM);
+        boolean groupRerouting = parameterService.getParameterValueAsBoolean(ProcurementCardRerouteDocumentsStep.class, KFSParameterKeyConstants.ProcurementCardParameterConstants.REROUTE_PCDO_DOCUMENTS_IND_PARM_NM);
         
```

Comment:
This method is outside of the scope of the Step 12 ticket, and is part of Step 13-16. This was only modified due to moving the constants to where they belong.